third_party/xla/ordered_set.h

```diff
@@ -56,12 +56,16 @@ class OrderedSet {
   // set.
   void Erase(T value) {
     auto it = value_to_index_.find(value);
-    assert(it != value_to_index_.end());
```

Comment:
I guess coverity was complaining about hitting this assert?

Reply:
> I guess coverity was complaining about hitting this assert?

So I am also guessing that Bibrak's modification is correct, right? We need to check this during runtime.

Reply:
`assert` is runtime but somehow perhaps the compiler/pre-processor removes it altogether... maybe

Reply:
*normally assert() is only compiled in during literal debug builds. Coverity uses release builds for analysis
So it is if this line doesn't exist